Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -12,157 +11,97 @@ To get started with the Mixed Reality Toolkit you will need:
-
-* Main Mixed Reality Toolkit Configuration
-* Camera Settings
-* Input System Settings
-* Boundary Visualization Settings
-* Teleporting Settings
-* Spatial Awareness Settings
-* Diagnostics Settings
-* Additional Services Settings
-* Input Actions Settings
-* Input Actions Rules
-* Pointer Configuration
-* Gestures Configuration
-* Speech Commands
-* Controller Mapping Configuration
-* Controller Visualization Settings
-
-As you can see there are lots of options available and more will come available as we progress through the beta.
-
-When you start a new project, we provide a default set of configurations with every option turned on, styled for a fully cross platform project.  These defaults are "Locked" to ensure you always have a common start point for your project and we encourage you to start defining your own settings as your project evolves.  For this we provide options to either:
-
-* Copy the defaults in to a new profile for you to start customizing it for your project
-* Start afresh with a brand-new profile.
-
-![](../External/ReadMeImages/MRTK_CopyCreateConfigurationOptions.png)
-
-When profiles are created by the MRTK, they are then placed in the following folder:
-
-> "Assets\MixedRealityToolkit-Generated\CustomProfiles"
-
-At each step in the configuration, you can choose to remove and create a new profile, or simply copy the existing settings and continue to customize:
-
-![](../External/ReadMeImages/MRTK_CopyProfileOptions.png)
-
-### **[For more information on customizing the Configuration Profiles](MixedRealityConfigurationGuide.md)**
-Please check out the [Mixed Reality Configuration Guide](MixedRealityConfigurationGuide.md)
+5. Hit play
+Press the play button to start the scene. You are now ready to start building your project, just start adding content and get building. Here are suggested next steps.
 
 ## Get building your project
 
-Now your project is up and running, you can start building your Mixed Reality project.  
-
-For more information on the rest of the toolkit, please check the following guides:
+Now your project is up and running, you can start building your Mixed Reality project. Here are some suggested next steps:
 
-* [Mixed Reality Configuration Guide](MixedRealityConfigurationGuide.md)
-* [Getting to know the Mixed Reality Toolkit Input System]() (Coming Soon)
-* [Customizing your controllers in the MRTK]() (Coming Soon)
-* [A walkthrough the UX components of the MRTK SDK]() (Coming Soon)
-* [Using Solvers to bind your objects together]() (Coming Soon)
-* [Creating interactions between the player and your project]() (Coming Soon)
-* [Configuration Profile Usage Guide]() (Coming Soon)
-* [Guide to building Registered Services]() (Coming Soon)
-* [Guide to Pointers documentation]() (Coming Soon)
+* Explore the [hand interaction examples scene](README_HandInteractionExamples.md) if you haven't already. Instructions to get started are above.
```

Comment:
>Explore the [hand interaction examples scene](README_HandInteractionExamples.md) if you haven't already. Instructions to get started are above. [](start = 2, length = 143)

I probably would omit this as a "next step" since it should have been already done :)